tck/README.md

```diff
@@ -188,14 +200,17 @@ public class RangePublisherTest extends PublisherVerification<Integer> {
 
 Notable configuration options include:
 
-* `maxElementsFromPublisher` – which should only be overridden in case the Publisher under test is not able to provide arbitrary length streams, e.g. it's wrapping a `Future<T>` and thus can only publish up to 1 element. In such case you should return `1` from this method. It will cause all tests which require more elements in order to validate a certain Rule to be skipped,
-* `boundedDepthOfOnNextAndRequestRecursion` – which should only be overridden in case of synchronous Publishers. This number will be used to validate if a
-`Subscription` actually solves the "unbounded recursion" problem (Rule 3.3).
+* `maxElementsFromPublisher` – which should only be overridden in case the Publisher under test is not able to provide
+  arbitrary length streams, e.g. it's wrapping a `Future<T>` and thus can only publish up to 1 element. In such case you
+  should return `1` from this method. It will cause all tests which require more elements in order to validate a certain
+  Rule to be skipped,
+* `boundedDepthOfOnNextAndRequestRecursion` – which should only be overridden in case of synchronous Publishers.
```

Comment:
This explanation does not really assist the reader in understanding what to set it to, so either that's described on that method, in which case we can point the reader to that, otherwise I'm afraid we need to elucidate the reader a bit more here.